### ISSUE TITLE
Fatal error method not found get_version() because of $dlm_download is null

### ISSUE THREAD

**Issue by tnchuntic** [download-monitor/download-monitor#627]
Fatal error in line 12 templates > content-download.php.
$dlm_download is null causing the page edit throwing fatal error.

```
If($dlm_download->get_version()->has_version_number()){
....
}
```

**Comment by tnchuntic** [comment]
Found temporary fix using if statement to check 
`if(!$dlm_download) return;`

**Comment by barrykooij** [comment]
Thank you, should be fixed in 4.4.4
